src/components/JsonSchemaValidator/Schemas/dbtUnified_0.1.2.js

```diff
@@ -604,5 +604,12 @@ export const Schema = {
       packageDefault: 'true',
       group: 'Warehouse and Tracker',
     },
+    snowplow__enable_screen_summary_context: {
+      type: 'boolean',
+      title: 'Enable Screen Summary Context',
+      longDescription: 'Flag to process the screen engagement information in the screen summary context on mobile events.',
+      packageDefault: 'false',
+      group: 'Contexts, Filters, and Logs',
+    },
```

Comment:
Duplicate the file then rename to 0.2.0 please, we now keep 1 schema per version

Reply:
I have added the missing vars to 0.2.0 as i didnt see this beforehand